oppia/oppia.py

```diff
@@ -0,0 +1,148 @@
+# coding: utf-8
+#
+# Copyright 2015 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This XBlock embeds an instance of Oppia in the OpenEdX platform."""
+
+import pkg_resources
+
+from eventtracking import tracker
+
+from xblock.core import XBlock
+from xblock.fields import Scope, Integer, String
+from xblock.fragment import Fragment
+
+
+class OppiaXBlock(XBlock):
+    """
+    An XBlock providing an embedded Oppia exploration.
+    """
+    _EVENT_NAME_EXPLORATION_LOADED = 'oppia.exploration.loaded'
+    _EVENT_NAME_EXPLORATION_COMPLETED = 'oppia.exploration.completed'
+    _EVENT_NAME_STATE_TRANSITION = 'oppia.exploration.state.changed'
+
+    # The display name of the component. Note that this is not editable in
+    # Studio.
```

Comment:
I thought this was just the internal title of the block in Studio ("Oppia Exploration" in your screenshot above)? I looked at some other XBlocks, and didn't see this field in their studio edit_html file.

The comment is here mainly to distinguish this field from the other ones.